tapir/coop/migrations/0001_initial.py

```diff
@@ -416,7 +416,7 @@ class Migration(migrations.Migration):
                 ("start_date", models.DateField(db_index=True)),
                 ("end_date", models.DateField(blank=True, db_index=True, null=True)),
                 (
-                    "owner",
+                    "share_owner",
```

Comment:
We can't update old migrations, because the live instance needs to go incrementally from the last migration it applied. This change needs to be in a new migration file.

Reply:
Yep, no idea how I landed here